pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
The sql_test.go file (along with the rest of the codebase) has been ported to use the new protobuf library. The wrappers for those live in the "google.golang.org/protobuf/types/known/wrapperspb" package, which is imported below. We should use that package instead.

Reply:
Good catch, thank you!